sdk/keyvault/tests.yml

```diff
@@ -16,5 +16,49 @@ jobs:
       AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
       AZURE_CLIENT_ID: $(net-keyvault-azure-client-id)
       AZURE_CLIENT_SECRET: $(net-keyvault-azure-client-secret)
-      AZURE_KEYVAULT_URL: $(net-keyvault-azure-keyvault-url)
+      AZURE_KEYVAULT_URL: $[coalesce('$(AZURE_KEYVAULT_NAME)', '$(net-keyvault-azure-keyvault-url)')]
```

Comment:
Unless you also set `AZURE_KEYVAULT_NAME` outside this block I don't think this will ever kick in.

Reply:
I think you are going to need to override `AZURE_KEYVAULT_URL` after you create the vault.

Reply:
This is the problem I was worried about, though I'm not sure how I'm going to overwrite it. Setting an environment variable in one step won't translate to another. Perhaps if I `export` it on macOS and linux, and `setx` on Windows?

Reply:
I don't think we want to set a global env variable. By turning this into a runtime evaluation via `$[...]` I think you can just set `net-keyvault-azure-keyvault-url` via `##vso[task.setvariable variable=net-keyvault-azure-keyvault-url]${Env:AZURE_KEYVAULT_NAME}`